pipeline/extract.py

```diff
@@ -85,20 +87,72 @@ def get_website_from_url(url: str) -> str:
     return website_url
 
 
-def scrape_from_html(html_content: bytes, url: str, product_id: int) -> dict:
+def scrape_pricing_process(html_content: bytes, url: str, product_id: int) -> dict:
+    """ Chooses which scraper to use based off of the URL """
+
+    website_url = get_website_from_url(url)
+
+    if "https://store.steampowered.com/" in url:
+        return scrape_from_steam_html(html_content, url, product_id)
+
+    if "https://www.amazon.co" in url:
+        return scrape_from_amazon_html(html_content, url, product_id)
+
+    logging.error(
+        "Cannot scrape that URL, since it's not an Amazon/Steam webpage.")
+    return
+
+
+def scrape_from_amazon_html(html_content, url, product_id):
     """ Scrapes from html to get a dictionary with the:
     - Product_ID 
     - product_name
     - original_price
     - discount_price
     - website 
-    """
+    for Amazon Product Pages."""
```

Comment:
Are you able to shorten this docstring? e.g. "Scrapes product, price and website information from Amazon."

Reply:
Done.